sklearn/model_selection/_search.py

```diff
@@ -552,59 +593,140 @@ def _fit(self, X, y, labels, parameter_iterable):
             pre_dispatch=pre_dispatch
         )(delayed(_fit_and_score)(clone(base_estimator), X, y, self.scorer_,
                                   train, test, self.verbose, parameters,
-                                  self.fit_params, return_parameters=True,
+                                  self.fit_params, return_parameters=False,
                                   error_score=self.error_score)
           for parameters in parameter_iterable
           for train, test in cv.split(X, y, labels))
 
+        out = np.array(out, dtype=np.float64)
```

Comment:
If you'd rather create an array of type object, then slice out of it (or do the same tuple unpacking) and astype it, we could... I just don't see why.